projects/packages/backup/src/js/components/Admin/index.js

```diff
@@ -180,7 +181,7 @@ const BackupSegments = ( { hasBackupPlan, connectionLoaded } ) => {
 
 const ReviewMessage = connectionLoaded => {
 	const [ restores ] = useRestores( connectionLoaded );
```

Comment:
Think we'll need to do similar work for restores, or it doesn't present the same issues?

Reply:
I put more attention in the backups endpoint because it was being called from 3 different places, but restores are only being called in this component, but yeah, I think it deserves a look to see how could we improve it.